docs/src/network-definition-spec.md

```diff
@@ -1,6 +1,10 @@
 # Network definition spec
 
+<<<<<<< Updated upstream
```

Comment:
Hi @wirednkod, this file have merge conflicts committed.

Reply:
Thank you @pepoviola